hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/bucket/TestBucketCachePersister.java

```diff
@@ -119,21 +120,36 @@ public void testPrefetchPersistenceCrash() throws Exception {
 
   @Test
   public void testPrefetchPersistenceCrashNegative() throws Exception {
-    long bucketCachePersistInterval = 3000;
+    long bucketCachePersistInterval = 5000;
```

Comment:
Actually, we should make BucketCachePersister a daemon thread. Then we could set this interval to Long.MAX_VALUE.